app/assets/images/payment_icons/seveneleven.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-seveneleven"><title id="pi-seveneleven">7-Eleven</title><defs><style>.cls-2{fill:#fff}.cls-3{fill:#008357}.cls-4{fill:#ed1c2e}</style></defs><path d="M35 0H3a3 3 0 00-3 3v18a3 3 0 003 3h32a3 3 0 003-3V3a3 3 0 00-3-3z" fill="#e8e8e8"/><path class="cls-2" d="M35 1a2 2 0 012 2v18a2 2 0 01-2 2H3a2 2 0 01-2-2V3a2 2 0 012-2h32"/><circle class="cls-3" cx="19" cy="12" r="9.85"/><path class="cls-2" d="M24.6 19a.7.7 0 01-.54.53h-2.85v.54h-4.16v-.54h-3.11a.69.69 0 01-.3-.17.81.81 0 01-.22-.43l-1.3-12.75a.68.68 0 01.45-.57h12.74a.74.74 0 01.56.44.91.91 0 010 .33z"/><path class="cls-4" d="M23.92 10.2a4.25 4.25 0 00-2.47 2.8H18a3.64 3.64 0 01.14-.47 7.78 7.78 0 012.07-3.27 10.58 10.58 0 012.87-2 3.62 3.62 0 01.55-.2c.09 0 .17-.06.28-.07v3.21"/><path d="M22.84 7.1a8.48 8.48 0 00-1.92 1.2 11.33 11.33 0 00-1.79 1.7h-4.72V7z" fill="#f58220"/><path class="cls-3" d="M20.6 13.26c-.11.81-.23 1.61-.38 2.42h-1.1c-.12-.81-.25-1.61-.36-2.41h.74c.1.6.17 1.21.28 1.79.11-.59.18-1.2.28-1.8h.51m-5 .49H15v.42h.57v.52H15v.48h.57v.5h-1.34v-2.42h1.33v.51m3 0H18v.42h.57v.52H18v.48h.57v.5h-1.29v-2.43h1.33v.51m3.51 0h-.58v.42h.57v.52h-.57v.48h.57v.5h-1.33v-2.43h1.34v.51m-6.3-.51h.74v1.93h.54v.5h-1.28v-2.43m7.27.27a.68.68 0 01.61-.34c.43 0 .51.31.51.93v1.56h-.74v-1.51c0-.22 0-.36-.17-.36s-.21.17-.21.45v1.42h-.74v-2.44h.74v.29m-1.17 3c.08 0 .15 0 .15-.1s-.07-.09-.13-.09h-.13v.19h.11m-.11.35h-.09v-.67H22c.15 0 .22.06.22.17a.16.16 0 01-.13.17l.18.28h-.11l-.16-.27h-.11v.27zm.11.14a.46.46 0 00.46-.45.45.45 0 00-.9 0 .44.44 0 00.42.45zm-.54-.49a.55.55 0 01.62-.47.54.54 0 01.47.47.56.56 0 01-.54.56.55.55 0 01-.54-.53z"/><path class="cls-4" d="M17.29 19.88v-1.16a19.93 19.93 0 01.24-2.78h3.53c-.13 1.31-.12 2.62-.12 3.94z"/></svg>
```

Comment:
The border color is slightly different from other icons: [video demonstration](https://screenshot.click/04-49-tn7na-no2w8.mp4)

Also, is it possible to use inline styles instead of using classes? For example

`<path class="cls-2" d="M24.6 19a.7.7 0 01-.54.53h-2.85v.54h-4.16v-.54h-3.11a.69.69 0 01-.3-.17.81.81 0 01-.22-.43l-1.3-12.75a.68.68 0 01.45-.57h12.74a.74.74 0 01.56.44.91.91 0 010 .33z"/>` 

would be come

`<path fill="#fff" d="M24.6 19a.7.7 0 01-.54.53h-2.85v.54h-4.16v-.54h-3.11a.69.69 0 01-.3-.17.81.81 0 01-.22-.43l-1.3-12.75a.68.68 0 01.45-.57h12.74a.74.74 0 01.56.44.91.91 0 010 .33z"/>`

Reply:
@tauthomas01 Thanks for the quick review. For the background color it seems Visa uses a slightly transparent background so I've applied he same transparency code on the seven eleven logo:

![image](https://user-images.githubusercontent.com/82835/89368270-12082f00-d716-11ea-96f2-878f9a5e54dd.png)

I've also removed inline styles from the SVG and rebased on master. Could you please check again 🙇